opentelekomcloud/resource_opentelekomcloud_cce_node_v3_test.go

```diff
@@ -267,3 +288,37 @@ resource "opentelekomcloud_cce_node_v3" "node_1" {
   }
 }
 `, OS_VPC_ID, OS_NETWORK_ID, OS_AVAILABILITY_ZONE, OS_KEYPAIR_NAME)
+
+var testAccCCENodeV3_ipParams = fmt.Sprintf(`
+resource "opentelekomcloud_cce_cluster_v3" "cluster_1" {
+  name         = "opentelekomcloud-cce"
+  cluster_type = "VirtualMachine"
+  flavor_id    = "cce.s1.small"
+  vpc_id       = "%s"
+  subnet_id    = "%s"
+
+  container_network_type = "overlay_l2"
+  authentication_mode    = "rbac"
+}
+
+resource "opentelekomcloud_cce_node_v3" "node_1" {
+  cluster_id        = opentelekomcloud_cce_cluster_v3.cluster_1.id
+  name              = "cce-node-1"
+  flavor_id         = "s2.xlarge.2"
+  availability_zone = "%s"
+  key_pair          = "%s"
+  root_volume {
+    size       = 40
+    volumetype = "SATA"
+  }
+
+  bandwidth_size = 100
+  sharetype      = "PER"
```

Comment:
Have you tested with `null`s?

Reply:
=== RUN   TestAccCCENodesV3_ipNulls
--- PASS: TestAccCCENodesV3_ipNulls (1939.93s)
PASS